src/boot/theme-provider.tsx

```diff
@@ -81,6 +81,38 @@ const themeSizes = (
 	}
 };
 
+const paletteExtension =
+	(): ThemeExtension =>
+	(t: any): any => {
```

Comment:
can this 't' cannot be typed ? also can you change the variable name from 't' to something else because we globally use 't' from Translation's T function